simple_planner:
  ros__parameters:
    use_manual_start: true

    # Cost function: "exponential" | "linear" | "inverse"
    cost_function_type: "exponential"

    # Cost function params
    inflation_radius: 0.5     # [m]
    alpha: 5.0                # [--]
    W: 20.0                   # [--]
    epsilon: 0.1              # [m] (usato con "inverse")
    lethal_cost: 1000000000.0 # “infinito” per ostacoli/unknown

    # A* weighting
    lambda_weight: 2.0
